From 763d003d702c52f033d428525e278c5fb8c6799f Mon Sep 17 00:00:00 2001
From: Paul Harrison <pharrison@mythtv.org>
Date: Tue, 11 Mar 2014 09:47:36 +0000
Subject: [PATCH 08/13] MainServer::HandleMusicTagGetImage: pass the ImageType
 by number not by name

The imagetype filename is translated which causes problems so safer to just
pass the ImageType value.
---
 mythtv/libs/libmythmetadata/musicmetadata.cpp |    2 +-
 mythtv/programs/mythutil/musicmetautils.cpp   |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/mythtv/libs/libmythmetadata/musicmetadata.cpp b/mythtv/libs/libmythmetadata/musicmetadata.cpp
index 777620a..7614190 100644
--- a/mythtv/libs/libmythmetadata/musicmetadata.cpp
+++ b/mythtv/libs/libmythmetadata/musicmetadata.cpp
@@ -1094,7 +1094,7 @@ QString MusicMetadata::getAlbumArtFile(void)
                     slist << QString("MUSIC_TAG_GETIMAGE %1 %2 %3")
                             .arg(Hostname())
                             .arg(ID())
-                            .arg(AlbumArtImages::getTypeFilename(albumart_image->imageType));
+                            .arg(albumart_image->imageType);
                     gCoreContext->SendReceiveStringList(slist);
                 }
             }
diff --git a/mythtv/programs/mythutil/musicmetautils.cpp b/mythtv/programs/mythutil/musicmetautils.cpp
index 27d1a82..1c3941e 100644
--- a/mythtv/programs/mythutil/musicmetautils.cpp
+++ b/mythtv/programs/mythutil/musicmetautils.cpp
@@ -101,7 +101,7 @@ static int ExtractImage(const MythUtilCommandLineParser &cmdline)
     }
 
     int songID = cmdline.toInt("songid");
-    ImageType type = AlbumArtImages::getImageTypeFromName(cmdline.toString("imagetype"));
+    ImageType type = (ImageType)cmdline.toInt("imagetype");
 
     MusicMetadata *mdata = MusicMetadata::createFromID(songID);
     if (!mdata)
-- 
1.7.10.2

